### PR TITLE
Fix support hardware serial

### PR DESCRIPTION
**Pull request by nomad605dis**
Small fix for hardware serial support by GPS module v2 on Neoway G7 chip.

I didn't touch the rest of the code because there will be a larger fix to the library soon.

### CODE REVIEW

File: examples/HardwareSerial/HardwareSerialGpsTest/HardwareSerialGpsTest.ino [hunk 1]
```diff
@@ -18,81 +18,89 @@ char latitudeBase60[MAX_SIZE_MASS];
...
+                  Serial.print(gps.getDay());
+                  Serial.print(gps.getMonth());
+                  Serial.print(gps.getYear());
+            */
+            break;
+        // ошибка данных
+        case GPS_ERROR_DATA:
+            Serial.println("GPS error data");
+            break;
+        // нет соединение со спутниками
```
Comment (yurybotov) [inline examples/HardwareSerial/HardwareSerialGpsTest/HardwareSerialGpsTest.ino:100]: // нет соединени**я** со спутниками

File: examples/HardwareSerial/HardwareSerialGpsTest/HardwareSerialGpsTest.ino [hunk 2]
```diff
@@ -18,81 +18,89 @@ char latitudeBase60[MAX_SIZE_MASS];
...
+        case GPS_ERROR_SAT:
+            Serial.println("GPS no connect to satellites!!!");
```
Comment (yurybotov) [inline examples/HardwareSerial/HardwareSerialGpsTest/HardwareSerialGpsTest.ino:102]: Serial.println("GPS is not connected to satellites!!!");

File: examples/SoftwareSerial/SoftwareSerialGpsTest/SoftwareSerialGpsTest.ino [hunk 3]
```diff
@@ -25,92 +25,89 @@ char latitudeBase60[MAX_SIZE_MASS];
...
+                  Serial.print(gps.getDay());
+                  Serial.print(gps.getMonth());
+                  Serial.print(gps.getYear());
+            */
+            break;
+        // ошибка данных
+        case GPS_ERROR_DATA:
+            Serial.println("GPS error data");
+            break;
+        // нет соединение со спутниками
```
Comment (yurybotov) [inline examples/SoftwareSerial/SoftwareSerialGpsTest/SoftwareSerialGpsTest.ino:107]: // нет соединени**я** со спутниками

File: examples/SoftwareSerial/SoftwareSerialGpsTest/SoftwareSerialGpsTest.ino [hunk 4]
```diff
@@ -25,92 +25,89 @@ char latitudeBase60[MAX_SIZE_MASS];
...
+        case GPS_ERROR_SAT:
+            Serial.println("GPS no connect to satellites!!!");
```
Comment (yurybotov) [inline examples/SoftwareSerial/SoftwareSerialGpsTest/SoftwareSerialGpsTest.ino:109]: Serial.println("GPS is not connected to satellites!!!");

**Review by yurybotov — APPROVED** [review]
Change 4 fixes and go forward

File: examples/HardwareSerial/HardwareSerialGpsTest/HardwareSerialGpsTest.ino (hunk 1 above)
Comment (igor605ds) [reply examples/HardwareSerial/HardwareSerialGpsTest/HardwareSerialGpsTest.ino]: Done

File: examples/HardwareSerial/HardwareSerialGpsTest/HardwareSerialGpsTest.ino (hunk 2 above)
Comment (igor605ds) [reply examples/HardwareSerial/HardwareSerialGpsTest/HardwareSerialGpsTest.ino]: Done

File: examples/SoftwareSerial/SoftwareSerialGpsTest/SoftwareSerialGpsTest.ino (hunk 3 above)
Comment (igor605ds) [reply examples/SoftwareSerial/SoftwareSerialGpsTest/SoftwareSerialGpsTest.ino]: Done

File: examples/SoftwareSerial/SoftwareSerialGpsTest/SoftwareSerialGpsTest.ino (hunk 4 above)
Comment (igor605ds) [reply examples/SoftwareSerial/SoftwareSerialGpsTest/SoftwareSerialGpsTest.ino]: Done

**Review by nomad605dis — COMMENTED** [review]
All changes accepted

**Pull request merged**